src/librustc_codegen_llvm/debuginfo/metadata.rs

```diff
@@ -1273,7 +1273,7 @@ impl EnumMemberDescriptionFactory<'ll, 'tcx> {
                         flags: DIFlags::FlagZero,
                         discriminant: Some(self.layout.ty.ty_adt_def().unwrap()
                                            .discriminant_for_variant(cx.tcx, i)
-                                           .val as u64),
+                                           .val as u128),
```

Comment:
What type is `val`? This cast might also be redundant.

Reply:
True, this is unnecessary. I removed `as u128`.